headers/functions/overlay29.h

```diff
@@ -274,6 +274,7 @@ bool CanAiMonsterMoveInDirection(struct entity* monster, enum direction_id direc
                                  bool* out_monster_in_target_position);
 bool ShouldMonsterRunAway(struct entity* monster);
 bool ShouldMonsterRunAwayVariation(struct entity* monster, undefined param_2);
+enum monster_relation GetRelationBetweenMonsters(struct entity* entity1, struct entity* entity2, bool param_3, bool param_4);
```

Comment:
Since params 3 and 4 are known, they could be named more descriptively.

Reply:
Done